Docs/Published/API-related/Chunking_API_Documentation.md

```
@@ -0,0 +1,369 @@
+    "propositions": {
+      "engine": ["heuristic", "spacy", "llm", "auto"],
+      "aggressiveness": [0, 1, 2],
+      "min_proposition_length": 15,
+      "prompt_profile": ["generic", "claimify", "gemma_aps"]
+    }
+  }
+}
+```
+
+Minimal OpenAPI schema (response) for stubbing
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)